omod/src/main/java/org/openmrs/module/referenceapplication/page/controller/LoginPageController.java

```diff
@@ -264,10 +268,17 @@ public String post(@RequestParam(value = "username", required = false) String us
 					}
 					
 					//If there is a single login location, default to that
+					boolean clientTimezoneProperty = false;
+					if(Context.isAuthenticated()){
+						clientTimezoneProperty = StringUtils.isBlank(Context.getAuthenticatedUser().getUserProperty("clientTimezone"));
+					}
 					if (sessionLocation == null) {
 						List<Location> loginLocations = appFrameworkService.getLoginLocations();
 						if (loginLocations.size() == 1) {
-							sessionLocation = loginLocations.get(0);
+							if(!ui.convertTimezones() || (ui.convertTimezones() && !clientTimezoneProperty)){
```

Comment:
Same here.

Reply:
@icrc-jfigueiredo  checkout very well this method convertTimezones(),it breaks every thing as per your ci logs

Reply:
@HerbertYiga that is because it depends from a version of [UiFramework ](https://github.com/openmrs/openmrs-module-uiframework/pull/66 )that is not merged yet, and that function, `convertTimezones(` ,is there.